src/System/Requirement/SessionsSecurityConfiguration.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\System\Requirement;
+
+/**
+ * @since 10.0.3
+ */
+class SessionsSecurityConfiguration extends AbstractRequirement
+{
+    public function __construct()
+    {
+        $this->title = __('Security configuration for sessions');
+        $this->description = __('Ensure security is enforced on session cookies.');
+        $this->optional = true;
+    }
+
+    protected function check()
+    {
+        $cookie_secure   = (bool)ini_get('session.cookie_secure');
+        $cookie_httponly = (bool)ini_get('session.cookie_httponly');
+        $cookie_samesite = ini_get('session.cookie_samesite');
+
+        $is_https_request = ($_SERVER['HTTPS'] ?? 'off') === 'on' || (int)($_SERVER['SERVER_PORT'] ?? null) == 443;
+
+        $validated = true;
+        if ($is_https_request && !$cookie_secure) {
+            $this->validation_messages[] = __('PHP directive "session.cookie_secure" should be set to "on" when GLPI can be accessed on HTTPS protocol.');
+            $validated = false;
+        }
+        if (!$cookie_httponly) {
+            $this->validation_messages[] = __('PHP directive "session.cookie_httponly" should be set to "on" to prevent client-side script to access cookie values.');
+            $validated = false;
+        }
+        if (strtolower($cookie_samesite) !== 'strict') {
+            $this->validation_messages[] = __('PHP directive "session.cookie_samesite" could be set to "Strict" to prevent cookie to be sent on cross-origin requests.');
+            $validated = false;
+        }
```

Comment:
It seems that `samesite=Strict` breaks the `oauthsso` plugin, which seems logical. So I think this check have to be removed (and probably a check that `samesite != strict` should be added in the plugin.